tensorrt_llm/_torch/speculative/interface.py

```
@@ -486,6 +579,20 @@ def populate_sampling_params_for_one_model(
         self.top_ps[:len(top_ps)].copy_(torch.tensor(
             top_ps, dtype=torch.float32, pin_memory=prefer_pinned()),
                                         non_blocking=True)
+        self.request_temperatures[:len(request_temperatures)].copy_(
+            torch.tensor(request_temperatures,
+                         dtype=torch.float32,
+                         pin_memory=prefer_pinned()),
+            non_blocking=True)
+        self.request_top_ks[:len(request_top_ks)].copy_(torch.tensor(
+            request_top_ks, dtype=torch.int32, pin_memory=prefer_pinned()),
+                                                        non_blocking=True)
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)